Update README.md

kaushikharsh630 opened the pull request.

kaushikharsh630 committed:
Update README.md

Varun-garg commented on README.md:
```diff
@@ -15,6 +15,8 @@ https://play.google.com/store/apps/details?id=com.varun.gbu_timetables
 
 - Support of both light and dark themes.
 
+-Soon aralrm feature is to come.
```
```suggestion
-Soon an alarm feature is to come.
```

Varun-garg commented on README.md:
```diff
@@ -29,4 +31,4 @@ Please read our [contributing guidelines](CONTRIBUTING.md) to get started.
 
-![Screenshots](Screenshots.jpg)
\ No newline at end of file
+![Screenshots](Screenshots.jpg)
```
what is the diff here?

Varun-garg reviewed (commented):
@kaushikharsh630  It would be pretty cool to have the feature. Looking forward to the change. + @amitkawasthi || @awasthi 

Please raise an issue wherever you need help and check out: https://github.com/Varun-garg/GBU_Timetables/blob/master/CONTRIBUTING.md

kaushikharsh630 commented:
There were many repos named "GBU time table" made the request to check
which one is active.
Don't worry about it I'm working on alarm feature and hopefully will be
able to do it.
By the way, is this the one on playstore?
Thankyou,
Harsh.n


On Fri, 2 Oct, 2020, 00:49 Varun Garg, <notifications@github.com> wrote:

> *@Varun-garg* commented on this pull request.
>
> @kaushikharsh630 <https://github.com/kaushikharsh630> It would be pretty
> cool to have the feature. Looking forward to the change. + @amitkawasthi
> <https://github.com/amitkawasthi> || @awasthi <https://github.com/awasthi>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Varun-garg/GBU_Timetables/pull/14#pullrequestreview-500647407>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQ7EL7ZUWUY4RP2J4WT6D6LSITI3HANCNFSM4SAHRZZQ>
> .
>

awasthi committed:
Release 74

This is major update as per guidelines of Google.
Some unsupported library  are updated.
Many bug-fixes.

Varun-garg commented:
pretty big change, nice!

Varun-garg commented:
@kaushikharsh630 awesome. I had put this one earlier in the play store, I think Amit sir will have a better idea to use which repo.
